script/validate-abl-import/test.bash

```diff
@@ -27,8 +27,10 @@ for input_file in "$my_dir"/*.input.json; do
     # Actual test goes here
     # -------------------------
     json=$(cat "$input_file")
-    node "$my_dir/../entry.js" transform-approved-books-data --data "$json" > "$output_file"
+    configured_books="{}"
```

Comment:
shouldn't there be tests that exercise the logic of diffing the books against the existing config?

Reply:
Thanks for the quick review! Yes, that completely slipped my mind. I added some additional test data for this. The smaller of the new data sets demonstrates what occurs when new versions are present, the other uses some real data to demonstrate that there are no new entries when all ABL versions are already present.